uiowa/uiowa
# Refactor uiowa_alerts to allow webmaster control with increased a11y and custom message.

- [x] Resolves #1275 
- [x] Resolves #1277 
- [x] Resolves #1278 
- [x] Resolves #1280 
- [x] Resolves #1323 
- [x] Resolves #1312 
- [x] Optimize cache/refresh settings
- [x] Show Hawk Alerts to authenticated users
- [x] ~~Enable in default config and update permission for webmasters~~

### Test
Remote: Inspect alerts already configured https://uiowa.dev.drupal.uiowa.edu/

Local: 
1. `blt dsa`
2. `blt frontend`
3. Go to admin -> config -> University of Iowa Alerts.
4. Make sure 'Display Hawk Alerts' is checked by default.
5. Change any config value.
6. Test that config values are self-explanatory, reflected on the frontend and are config-ignored.
